src/index.ts

```diff
@@ -56,7 +56,7 @@ async function getDevtoolsPanel(
     // Wait for UI.viewManager to become available
     await devtools.waitForFunction(
       /* istanbul ignore next */
-      () => 'UI' in window,
+      () => 'viewManager' in window.UI,
```

Comment:
Is `window.UI` _always_ defined, or should this be:

```suggestion
      () => 'UI' in window && 'viewManager' in window.UI,
```

Reply:
It has been, but I guess it doesn't hurt to check for both